pysat/tests/test_instrument.py

```diff
@@ -1070,10 +1047,10 @@ def test_fid_data_padding_removal(self):
 #
 # ------------------------------------------------------------------------------
 
-class TestDataPaddingbyFileXarray():
+class TestDataPaddingbyFileXarray(TestDataPaddingbyFile):
```

Comment:
these tests were being skipped

Reply:
develop-3: https://coveralls.io/builds/30322206/source?filename=pysat%2Ftests%2Ftest_instrument.py#L1073
this branch: https://coveralls.io/builds/30334382/source?filename=pysat%2Ftests%2Ftest_instrument.py#L1050